docs/admin/maintenance/upgrade/upgrade.mdx

```diff
@@ -5,25 +5,25 @@ title: 'Upgrade'
 description: '🔄 Keep your setup up to date with the latest features!'
 ---
 
-import Tabs from '@theme/Tabs'
-import TabItem from '@theme/TabItem'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-This guide provides steps to upgrade OpenCloud for both [docker](docs/admin/getting-started/container/docker.md) and [docker compose](docs/admin/getting-started/container/docker-compose/docker-compose.md)
+This guide provides steps to upgrade OpenCloud for both [docker](docs/admin/getting-started/container/docker.md) and
+[docker compose](docs/admin/getting-started/container/docker-compose/docker-compose.md)
 
 ### 1. Stop OpenCloud
 
 Stop the currently running OpenCloud instance:
 
 <Tabs groupId="deployment">
+  {/* prettier-ignore */}
   <TabItem value="docker" label="docker">
-    ```Shell
-    docker stop opencloud
-    ```
-  </TabItem>
+        ```shell
+        docker stop opencloud
+        ```
+    </TabItem>
   <TabItem value="docker-compose" label="docker compose">
-    ```Shell
-    docker compose stop
-    ```
+    ```shell docker compose stop ```
```

Comment:
The line breaks need to stay as they were for this to display. This also happens on multiple locations.

Reply:
Those are bad news mdx seems to be not quite supported